src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -696,6 +696,14 @@ function InitializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
   process.env.GCLOUD_PROJECT = frb.projectId;
   process.env.FUNCTIONS_EMULATOR = "true";
 
+  const nodeVersion = parseVersionString(process.versions.node);
+  if (nodeVersion.major >= 10) {
+    const triggerId = frb.triggerId || "";
+    process.env.FUNCTION_TARGET = triggerId.replace(/-/g, ".");
```

Comment:
I think if we are going to define some of these variables we should define all of them.  Here's what I see in production:

Function:
```js

exports.firestoreFn = functions.firestore.document('somecol/doc').onWrite(() => {
    console.log('FUNCTION_TARGET', process.env.FUNCTION_TARGET);
    console.log('FUNCTION_SIGNATURE_TYPE', process.env.FUNCTION_SIGNATURE_TYPE);
    console.log('K_SERVICE', process.env.K_SERVICE);
    console.log('K_REVISION', process.env.K_REVISION);
    console.log('PORT', process.env.PORT);

    return true;
});
```

Logs
```
2019-08-29T20:52:01.413Z ? firestoreFn: FUNCTION_TARGET firestoreFn
2019-08-29T20:52:01.413Z ? firestoreFn: FUNCTION_SIGNATURE_TYPE event
2019-08-29T20:52:01.413Z ? firestoreFn: K_SERVICE firestoreFn
2019-08-29T20:52:01.413Z ? firestoreFn: K_REVISION 2
2019-08-29T20:52:01.413Z ? firestoreFn: PORT 8080
```

I think for `K_REVISION` we can always do `1` and for `PORT` we can do `80` since locally we always serve over `http` (not really sure why it's `8080` in prod to be honest).

Reply:
Thanks for all helpful feedback! :star: Will try to finish this as soon as I get time to. If someone else steps up and do it, feel free to. :dancer: